src/main/java/blusunrize/immersiveengineering/common/items/RailgunProjectiles.java

```diff
@@ -129,5 +134,36 @@ public Entity getProjectile(@Nullable Player shooter, ItemStack ammo, Entity def
 				return defaultProjectile;
 			}
 		});
+
+		if(ModList.get().isLoaded("tetra"))
+		{
+			// Salvaged Beam
+			RailgunHandler.registerProjectile(() -> Ingredient.of(ItemBeam.instance), new RailgunHandler.StandardRailgunProjectile(40, 1.5)
+					.setColorMap(new RailgunRenderColors(0x383838, 0x383838, 0x383838, 0x383838, 0x2f2f2f, 0x252525)));
+
+			// Extractor
+			RailgunHandler.registerProjectile(() -> Ingredient.of(ChthonicExtractorBlock.item, ChthonicExtractorBlock.usedItem), new RailgunHandler.IRailgunProjectile()
```

Comment:
You're referencing non-API classes in tetra here. I don't know if they have an API that exposes these items, if they don't you should use `RegistryObject`s to access them rather than internal classes that may be renamed/changed at any point.